pimd/pim_nht.c

```diff
@@ -106,7 +106,7 @@ static struct pim_nexthop_cache *pim_nexthop_cache_add(struct pim_instance *pim,
 }
 
 static struct pim_nexthop_cache *pim_nht_get(struct pim_instance *pim,
-					     struct prefix *addr)
+					     pim_addr *addr)
```

Comment:
Everywhere else, we pass `pim_addr` as value, not pointer — let's make it consistent please.

Reply:
Fixed